minorminer/utils/parallel_embeddings.py

```diff
@@ -334,14 +334,14 @@ def find_sublattice_embeddings(
         T: The target graph in which to embed. If
             raster_embedding is not None, the graph must be of type zephyr,
             pegasus, or chimera and constructed by dwave_networkx.
-        tile: A mask applied to the target graph ``T`` defining a restricted
-            space in which to search for embeddings; the tile family
-            should match that of ``T``. If the tile is not
-            provided, it is generated as a fully yielded square sublattice
-            of ``T``, with ``m=sublattice_size``.
-            If ``tile==S``, ``embedder`` can be ignored since a 1:1 mapping is
-            necessary on each subgraph and easily verified by checking the
-            mask is complete.
+        tile: A dwave_networkx compatible mask applied to the target graph ``T``
```

Comment:
Maybe clarify what's "dwave_networkx compatible mask"?

Reply:
Easiest to say 'Chimera, Pegasus or Zephyr graph'

Reply:
Although that doesn't capture the subtlety that they have to be generated by dwave_networkx code